client/me/purchases/remove-purchase/index.jsx

```diff
@@ -475,15 +486,21 @@ class RemovePurchase extends Component {
 }
 
 export default connect(
-	( state, { selectedSite } ) => ( {
-		isDomainOnlySite: selectedSite && isDomainOnly( state, selectedSite.ID ),
-		isAutomatedTransferSite: selectedSite && isSiteAutomatedTransfer( state, selectedSite.ID ),
-		isChatAvailable: isHappychatAvailable( state ),
-		isChatActive: hasActiveHappychatSession( state ),
-		purchasesError: getPurchasesError( state ),
-		precancellationChatAvailable: isPrecancellationChatAvailable( state ),
-		userId: getCurrentUserId( state ),
-	} ),
+	( state, { purchase, selectedSite } ) => {
+		const isJetpack = purchase && isJetpackPlan( purchase );
+		return {
+			isDomainOnlySite: purchase && isDomainOnly( state, purchase.siteId ),
+			/* We know Jetpack sites are not Atomic */
```

Comment:
I find this comment's wording a bit confusing, since e.g. IIRC `isJetpackSite( state, atomicSiteId ) === true`. So maybe make it a bit more plans specific since that's what we're after here? E.g.

`If the site is on a Jetpack plan, we know it's not Atomic`

Reply:
Atomic are Jetpack _sites_, but an Atomic site will never have a Jetpack _plan_, which is what this check is expressing. I'll improve the comment.